engine/lib/luajit-ffi-gen/src/impl_item/type_info.rs

```diff
@@ -5,6 +5,7 @@ const RUST_TO_LUA_TYPE_MAP: &[(&str, &str)] = &[
     ("DVec2", "Vec2d"),
     ("Vec2", "Vec2f"),
     ("Vec3", "Vec3f"),
+    ("Vec4", "Vec4f"),
 ];
 
 // TODO: find out different way to mark types as copyable
```

Comment:
Do you mind adding the remaining vector types as well as Box3? I did it in my Rapier PR, but I think it would be good for you to add it here, as you're merging this soon: https://github.com/Limit-Theory-Redux/ltheory/pull/92/files#diff-3182aa17288a8515eed368a0d87ab758a0b02ab3a4f9c6f55b743351a84a13d1

Reply:
Done except changes in `COPY_TYPES` array - only ***Vec2** types can be copuable